app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -520,7 +520,6 @@ class PasswordStore : BaseGitActivity() {
         val intent = Intent(this, SelectFolderActivity::class.java)
         val fileLocations = values.map { it.file.absolutePath }.toTypedArray()
         intent.putExtra("Files", fileLocations)
-        intent.putExtra(REQUEST_ARG_OP, "SELECTFOLDER")
         registerForActivityResult(StartActivityForResult()) { result ->
```

Comment:
Not entirely sure about the line above, everything else looks fine.

Reply:
The value is not being used anywhere and this is the only place where the string 'SELECTFOLDER' is used.